package/tests/conftest.py

```diff
@@ -23,14 +23,31 @@
 
 from kedro_viz.api import apps
 from kedro_viz.data_access import DataAccessManager
+from kedro_viz.integrations.kedro.sqlite_store import SQLiteStore
 from kedro_viz.server import populate_data
 
+BUCKET_NAME = "test-bucket"
+
 
 @pytest.fixture
 def data_access_manager():
     yield DataAccessManager()
 
 
+@pytest.fixture
+def session_store_mock(tmp_path):
+    # Create a MagicMock object for SQLiteStore
+    session_store_mock = mock.MagicMock(spec=SQLiteStore)
+
+    # Create a PropertyMock object for the location attribute
+    location_mock = mock.PropertyMock(return_value=Path(tmp_path / "session_store.db"))
+    type(session_store_mock).location = location_mock
+
+    session_store_mock.sync = mock.MagicMock()
+
+    yield session_store_mock
```

Comment:
Why do you need to mock this all out? Why not use a real `BaseSessionStore("dummy_path", "dummy_session_id")`?

Reply:
works great!

Reply:
BaseSessionStore("dummy_path", "dummy_session_id")
this works but i get code coverage error in server.py (lines 40-42 where it checks for if(sessionStore?) 

should I revert it to back to SQLiteStore mock or shld i just do #pragma:ignore. 
I wouldn't want to do ignore.